### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by re-fox**
Uploading copy of Vovalex ransomware for new capa rules

### CODE REVIEW

**Comment by williballenthin** [comment]
i think this file is empty?

image

**Comment by re-fox** [comment]
Strange, I'm using the web interface, let me try again.

**Comment by williballenthin** [comment]
image
there it is

**Comment by williballenthin** [comment]
minor concern for the size/complexity of this binary. 5MB is fairly large, and weighs down the size of this repo and runtime of testing/linting.

in your opinion, is it feasible to find a smaller sample, or do you expect that all D samples will be fairly large?

**Comment by re-fox** [comment]
Good question.  I'll dig and see what I can find.

**Comment by re-fox** [comment]
I have a sample that is much smaller (475k), I'll close out this PR and open another one with the new binary.  I'll also adjust the meta in `capa-rules/compiler/d/compiled-with-dmd.yml` to reflect the change.
